iocage_lib/ioc_plugin.py

```diff
@@ -149,18 +148,18 @@ def fetch_plugin(self, _json, props, num, accept_license):
                     paths=plugin_devfs_paths,
                     includes=plugin_devfs_includes
                 )
-            jail_name, jaildir, _conf, repo_dir = self.__fetch_plugin_create__(
-                props, jail_name)
-            location = f"{self.iocroot}/jails/{jail_name}"
-            self.__fetch_plugin_install_packages__(jail_name, jaildir, conf,
-                                                   pkg, props, repo_dir)
-            self.__fetch_plugin_post_install__(conf, _conf, jaildir, jail_name)
-        except Exception as e:
+            jaildir, _conf, repo_dir = self.__fetch_plugin_create__(props)
+            self.__fetch_plugin_install_packages__(
+                jaildir, conf, pkg, props, repo_dir
+            )
+            self.__fetch_plugin_post_install__(conf, _conf, jaildir)
+        except BaseException as e:
+            self.keep_jail_on_failure = False
```

Comment:
I am sorry i am not sure if i follow this 🤔 Why do we hardcode this value here on an exception ?
Do we want to destroy plugins always on failure ?

Reply:
Debug :D I can't believe I forgot it.

Reply:
Happens to the best of us ;)